src/components/activity-page/embeddable.tsx

```diff
@@ -93,10 +92,10 @@ export const Embeddable: React.ForwardRefExoticComponent<IProps> = forwardRef((p
     qComponent = <div>Content type not supported</div>;
   }
 
-  // The following conditional prevents teacher edition containers from being rendered 
-  // when not in teacher edition mode. LARA handles this differently by using a mutation observer 
-  // (see https://github.com/concord-consortium/lara/blob/master/app/views/plugins/_show.haml). 
-  // It would be better to do that here as well if we update Activity Player to not use direct 
+  // The following conditional prevents teacher edition containers from being rendered
+  // when not in teacher edition mode. LARA handles this differently by using a mutation observer
+  // (see https://github.com/concord-consortium/lara/blob/master/app/views/plugins/_show.haml).
+  // It would be better to do that here as well if we update Activity Player to not use direct
```

Comment:
```suggestion
  // It would be better to do that here as well now that Activity Player no longer has direct
```

Reply:
I checked this and it turns out not all all direct dependency on teacherEditionMode has been removed. For this specific case line 89 is the key:
`qComponent = teacherEditionMode ? <EmbeddablePlugin embeddable={embeddable} pluginsLoaded={pluginsLoaded} /> : undefined;`

So this comment before Kirks corrections is still correct.